packages/lib-oer/test/string.test.ts

```diff
@@ -69,7 +69,27 @@ describe("string", () => {
         expect,
       }) => {
         const value = schema.serialize("12345678901κ")
-        expect(value).toMatchSnapshot()
+        expect(value).toMatchInlineSnapshot(`
+          {
+            "success": true,
```

Comment:
TODO: should refuse to serialize a value of character length 12 but byte length of 13